test/e2e/serial/tests/configuration.go

```diff
@@ -70,7 +70,7 @@ import (
 	serialconfig "github.com/openshift-kni/numaresources-operator/test/e2e/serial/config"
 )
 
-var _ = Describe("[serial][disruptive][slow] numaresources configuration management", Serial, func() {
+var _ = Describe("[serial][disruptive][slow] numaresources configuration management", Serial, Label("disruptive", "slow"), func() {
```

Comment:
if we'd like to proceed with the labels as the preferred way we should enable interacting with them using the binary, like this: https://github.com/openshift-kni/numaresources-operator/pull/951/files (the shell script part only)

Reply:
yes, this can be done import your change in this PR or deferring the script changes to another PR. I'm fine both ways.